src/Stack/SetupCmd.hs

```diff
@@ -102,9 +102,11 @@ setup SetupCmdOpts{..} wantedCompiler compilerCheck mstack = do
         , soptsSetupInfoYaml = scoSetupInfoYaml
         , soptsGHCBindistURL = scoGHCBindistURL
         , soptsGHCJSBootOpts = scoGHCJSBootOpts ++ ["--clean" | scoGHCJSBootClean]
+        , soptsBuildGHC = configCompilerGitBuild
         }
     let compiler = case wantedCompiler of
             WCGhc _ -> "GHC"
+            WCGhcGit{} -> "GHC"
```

Comment:
I think it's worth saying `"GHC (built from Git)"` or similar

Reply:
Done.